libpod/container_internal_linux.go

```diff
@@ -835,13 +835,13 @@ func (c *Container) checkpointRestoreLabelLog(fileName string) error {
 
 	logFile, err := os.OpenFile(dumpLog, os.O_CREATE, 0600)
 	if err != nil {
-		return errors.Wrapf(err, "failed to create CRIU log file %q", dumpLog)
+		return err
```

Comment:
I think the CRIU log file bit of the error might be useful here.

Reply:
fixed